packages/react-dom/src/events/ReactDOMEventListener.js

```diff
@@ -124,10 +123,10 @@ export function trapBubbledEvent(topLevelType, handlerBaseName, element) {
   if (!element) {
     return null;
   }
-  return EventListener.listen(
-    element,
+  element.addEventListener(
```

Comment:
This is only equivalent if `element` always has `addEventListener`. The original code in fbjs checked for it.

Are we 100% certain it always exists?